bootstrap.sh

```diff
@@ -1,20 +1,58 @@
 #!/bin/bash
-#This bootstrap script installs Docker and Jenkins
-
-# The docker part could also be accomplished using: sudo apt install docker && sudo systemctl start docker && sudo systemctl enable docker
-curl -fsSL https://get.docker.com -o get-docker.sh
-sh get-docker.sh
-
-cd $HOME
-sudo wget -O /etc/yum.repos.d/jenkins.repo https://pkg.jenkins.io/redhat-stable/jenkins.repo
-sudo rpm --import https://pkg.jenkins.io/redhat-stable/jenkins.io.key
-yum install jenkins
-sudo systemctl start jenkins.service
-sudo systemctl enable jenkins.service
+#This bootstrap script installs Git, configures it, and installs Docker and runs a Jenkins container.
 
+# Git installation
+wget https://github.com/git/git/archive/v2.29.1.tar.gz -O git.tar.gz
 tar -zxf git.tar.gz
+# The next line cds to the last created folder, whuch is the extracted files' folder
 cd  "$(\ls -1dt ./*/ | head -n 1)"
-make configure
 ./configure --prefix=/usr/local
-sudo make install
+make install
+
+#Git initial setup:
+git config --global user.name "Marom Goldshmidt"
```

Comment:
Hard coded values are meh.
Accept them as arguments.
<ake sure you are getting them from the user, else raise exception.

Reply:
```suggestion
git config --global user.name $1
```